ledger/ledger-api-client/src/main/scala/com/digitalasset/ledger/client/services/admin/PartyManagementClient.scala

```diff
@@ -117,6 +130,9 @@ final class PartyManagementClient(service: PartyManagementServiceStub)(implicit
       )
       .map(_.partyDetails.getOrElse(sys.error("No PartyDetails in response.")))
       .map(PartyManagementClient.details)
+      // TODO(https://github.com/DACH-NY/canton/issues/16401): remove the sleep call once we have a
```

Comment:
A non-flaky workaround could use this mechanism:
https://github.com/DACH-NY/canton/blob/e015fd5b3636273e28e8a761df50289906973e63/enterprise/ledger-test-tool/suites/src/main/scala/com/daml/ledger/api/testtool/infrastructure/participant/SingleParticipantTestContext.scala#L236C12-L236C12

Reply:
Good to know, thanks! I'll keep the sleeps for now as I'd like to merge this asap, but I'll consider moving to polling int a further PR.